packages/relay/src/lib/services/cacheService/cacheService.ts

```diff
@@ -321,4 +325,108 @@ export class CacheService {
     // fallback to internal cache in case of Redis error
     await this.internalCache.clear();
   }
+
+  /**
+   * Increments the value of a key in the cache by the specified amount.
+   * @param key - The key to increment.
+   * @param amount - The amount to increment by.
+   * @param callingMethod - The name of the calling method.
+   * @param requestIdPrefix - The optional request ID prefix.
+   * @returns {Promise<number>} A Promise that resolves with the new value of the key after incrementing.
+   */
+  public async incrBy(key: string, amount: number, callingMethod: string, requestIdPrefix?: string): Promise<number> {
```

Comment:
Let's make `requestIdPrefix` a mandatory argument. The Request ID is essential for tracing issues in the logs in GPC.

Additionally, I recommend passing the original `requestId` and then using the `formatter.formatRequestIdMessage()` method to ensure consistent formatting. It can be confusing for developers to determine when to pass the formatted `requestId` versus the original. Many logs I've seen in the past contained only the raw `requestId` due to this small detail, which appears unprofessional and inconsistent. Just to point it out, the `formatRequestIdMessage()` method can format an unformatted `requestId` or return the formatted `requestId` if it is already formatted. This approach provides developers with a flexible option for what to pass in, ensuring that the result will always be a formatted `requestId`. By adopting this method, logs will be consistent and synchronized with regard to `requestId`.

Let's tryna employ this logic where possible!

Reply:
This is out-of-scope for this PR, a good place to handle this is most likely #2867, since it will do changes to this argument in all methods.

Reply:
I have synced this with Kosi, so she will handle it there 🙌

Reply:
Great! Thanks for bringing this to her attention!